### PR TITLE
12.0 dashboard devang

### PR DESCRIPTION
**Pull request by devang-dreambits**
This PR contains changes made in new dashboard form marketplace base.

### CODE REVIEW

File: dbt_marketplace_base/models/base_template.py [hunk 1]
```diff
@@ -27,35 +30,87 @@ class DbtMarketplaceBase(models.Model):
...
-            self.log(self.child_class_name)
-            self.log(self.child_class_id)
-            self.log("Child class found")
-            self.env[self.child_class_name].browse(
-                self.child_class_id).with_delay().fetch_orders()
+        try:
+            self.log("Inside fetch_orders for marketplace")
+            self.log(self.enable_order_fetching)
+            if self.enable_order_fetching and self.child_class_name and self.child_class_id:
+                self.log("Function calling starts")
```
Comment (karan-dreambits) [inline dbt_marketplace_base/models/base_template.py]: Too many log statements are added just for debugging .. Cannot merge this PR with this debug statements

File: dbt_marketplace_base/views/dbt_instance_view.xml [hunk 2]
```diff
@@ -16,19 +16,9 @@
                                 <field name="name" />
                             </h1>
                         </div>
-                       <group string="Option" attrs="{'invisible':[('connected','=',False)]}">
-                        </group>
                         <group string="Connection">
-                            <!-- <field name="dbt_test"/> -->
                             <field name="active"/>
```
Comment (karan-dreambits) [inline dbt_marketplace_base/views/dbt_instance_view.xml:30]: other fields of base_template needs to be added.
Also can we think of some way to give link to connected Instance (Not only bol but for any other projects too)

**Review by karan-dreambits — CHANGES_REQUESTED** [review]

File: dbt_marketplace_base/views/dbt_instance_view.xml (hunk 2 above)
Comment (karan-dreambits) [reply dbt_marketplace_base/views/dbt_instance_view.xml:30]: Do comment your thoughts on how we can show linked instance before actually working on it @devang-dreambits

Comment (devang-dreambits) [reply dbt_marketplace_base/views/dbt_instance_view.xml:30]: Opening an action form a button can help us. But we need 1 more field for saving action of the connector class....ex 

'child_action_id': self.env.ref('dbt_bol_connector.action_bol_instances').id,

**Review by karan-dreambits — APPROVED** [review]

**Pull request merged**